src/CompanyService.Validation/Office/CreateOfficeRequestValidator.cs

```diff
@@ -4,15 +4,18 @@
 
 namespace LT.DigitalOffice.CompanyService.Validation.Office
 {
-    public class CreateOfficeRequestValidator : AbstractValidator<CreateOfficeRequest>, ICreateOfficeRequestValidator
+  public class CreateOfficeRequestValidator : AbstractValidator<CreateOfficeRequest>, ICreateOfficeRequestValidator
+  {
+    public CreateOfficeRequestValidator()
     {
-        public CreateOfficeRequestValidator()
-        {
-            RuleFor(request => request.City)
-                .NotEmpty();
+      RuleFor(request => request.Name.Trim())
```

Comment:
if name is null you will have nullRefExc

Reply:
i fixed it in this PR
https://github.com/LT-Students/DigitalOffice-CompanyService/pull/86